examples/selective_run/gantry.def.yml

```diff
@@ -0,0 +1,48 @@
+version: "0"
+
+services:
+  active_service:
+    image: alpine
+    command: "echo \"Step active_service\""
+    depends_on:
+      - move_data_to_active_service
+
+  new_service:
+    image: alpine
+    command: "echo \"Step new_service\""
```

Comment:
I think to be more realistic as services this should be some long running/infinite loop

Reply:
Done